test/amqp.js

```diff
@@ -46,8 +46,8 @@ describe('AMQP', function() {
       amqp.connect(done);
     });
     it('should declare your queue, and bind it', function(done) {
-      var amqpLibMock = require('./amqplibmock')();
-      var mockedAMQP = SandboxedModule.require('../../amqp', {
+      var amqpLibMock = require('./lib/amqplibmock')();
```

Comment:
wonder if the mock file should be in `/test/mocks` or something like that instead? leaving `/test/lib` for testing things in the main `/lib` dir? While you are at it, it might be worth moving the `/test/config*` files into a '/test/configs` dir too.